backend/app/Http/Controllers/TransactionController.php

```diff
@@ -25,4 +26,88 @@ public function show(Transaction $transaction)
     {
         return TransactionResource::make($transaction);
     }
+
+    public function store(TransactionPostRequest $request)
+    {
+        function processAccounts($payload)
```

Comment:
move function outside this `store` function

it seems this is only for transaction processes so make it private